genie/database_to_staging.py

```diff
@@ -189,23 +190,47 @@ def redact_phi(
     # Moved to cannotReleaseHIPAA and withheld because the HIPAA
     # years would change every single year. (e.g. <1926, >1998 would be
     # inaccurate every year)
+    complete_redact = pd.Series([False] * len(clinicaldf))
     for col in interval_cols_to_redact:
-        to_redact, to_redactpeds = _to_redact_interval(clinicaldf[col])
```

Comment:
To move forward

```
    to_redact = (col_int > phi_cutoff) | contain_greaterthan
```

This is of concern... so lets talk through this in a meeting, BUT...

89*365 = 32485

AND when you are 32485 days old, you are 89.  But if we are looking for > 32485, we are actually including the first day of 89.  So... should we be filtering out anything >=32485

Reply:
Confirmed with xindi that we don't use `equal to` check: https://sagebionetworks.jira.com/browse/GEN-1890?focusedCommentId=248144